platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -236,6 +245,20 @@ optional<std::pair<Response, uint64_t>> OfflineDatabase::getResource(const Resou
     return std::make_pair(response, size);
 }
 
+optional<int64_t> OfflineDatabase::hasResource(const Resource& resource) {
```

Comment:
Actually, I take that back -- it's probably better not to bump the accessed value in the situations where `hasResources` is called. It's a very weak form of access and probably shouldn't effect the eviction priority.